### PR TITLE
feat: document personal data storage options

### PR DESCRIPTION
**Pull request by alnr**

### CODE REVIEW

File: docs/security-compliance/personal-data-location.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+id: personal-data-location
+title: Personal data location and homing
+sidebar_label: Personal data storage
+---
+
+Ory Network is globally distributed for maximum resilience and minimum latency. Check out our
+[blog post](https://www.ory.sh/global-identity-and-access-management-multi-region/) for details on how this works.
+
+As discussed in that blog post, replicating personal data (also known as personally identifiable data or PII) across the globe is
+prohibited or made difficult by legislation like GDPR and would your user's control over their data.
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:11]: >would your user's control over their data.

I think something is missing here?

File: docs/security-compliance/personal-data-location.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Ory Network solves this apparent dilemma by separating personal data from operational data. Personal data is any data concerning
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:13]:
```suggestion
Ory Network solves this by separating personal data from operational data. Personal data is any data concerning
```

keep it short & concise 🙏

File: docs/security-compliance/personal-data-location.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+id: personal-data-location
+title: Personal data location and homing
+sidebar_label: Personal data storage
+---
+
+Ory Network is globally distributed for maximum resilience and minimum latency. Check out our
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:7]:
```suggestion
Ory Network is globally distributed for maximum resilience and minimum latency. Check out the
```

File: docs/security-compliance/personal-data-location.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+As discussed in that blog post, replicating personal data (also known as personally identifiable data or PII) across the globe is
+prohibited or made difficult by legislation like GDPR and would your user's control over their data.
+
+Ory Network solves this apparent dilemma by separating personal data from operational data. Personal data is any data concerning
+your users and identities managed by Ory: think email addresses, names, phone numbers, device and location data, emails, etc.
+Operational data is everything else: think "is this session or access token valid?" or "does the user with this anonymized ID have
+permission to access file X?"
+
+Ory Network globally replicates and makes available all operational data, giving you access with low latency based on the
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:18]:
```suggestion
Ory Network globally replicates and makes all operational data available, giving you access with low latency based on the
```

File: docs/security-compliance/personal-data-location.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+geographical location of where the request originates.
+
+Personal data, on the other hand, is always homed to a particular region and stored only there. For users from the EU, their
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:21]:
```suggestion
Personal data in Ory Network is always homed to a particular region and stored only there. For users from the EU, their
```

File: docs/security-compliance/personal-data-location.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+personal data is stored on servers physically located in the EU. For users from the US, their personal data is stored in the US.
+You get the idea.
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:23]:
```suggestion
```

File: docs/security-compliance/personal-data-location.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+Ory Network handles this distinction for you transparently. You don't have to do anything to opt-in. Whatever data you query from
+wherever you are located, Ory Network will combine any globally-replicated operational data with regional personal data for you
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:26]:
```suggestion
wherever you are located, Ory Network will combine any globally replicated operational data with regional personal data for you
```

File: docs/security-compliance/personal-data-location.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+automatically, reaching out behind the scenes to the region where that personal data is stored.
+
+## Selecting a storage location for personal data
+
+When you [create a new Ory Network project](https://console.ory.sh/projects/create) in the Ory Console, you are given a choice of
+personal data storage location for your new project. Any personal data if your users and identities will be stored that region
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:32]:
```suggestion
personal data storage location for your new project. Any personal data of your users and identities will be stored just in that region
```

File: docs/security-compliance/personal-data-location.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+only (replicated within that region for availability and durability of course). Note that this choice is permanent for that
```
Comment (vinckr) [inline docs/security-compliance/personal-data-location.md:33]:
```suggestion
, replicated within that region for availability and durability. Note that this choice is permanent for that
```

File: docs/security-compliance/personal-data-location.md (hunk 1 above)
Comment (jonas-jonas) [inline docs/security-compliance/personal-data-location.md:11]: I think there is a word missing here? At least to me, the would doesn't make sense?

**Review by vinckr — CHANGES_REQUESTED** [review]

File: docs/security-compliance/personal-data-location.md (hunk 1 above)
Comment (vinckr) [reply docs/security-compliance/personal-data-location.md:11]: ah you were faster ;-)

File: docs/security-compliance/personal-data-location.md (hunk 4 above)
Comment (alnr) [reply docs/security-compliance/personal-data-location.md:18]: This sublty changes the meaning.

File: docs/security-compliance/personal-data-location.md (hunk 1 above)
Comment (alnr) [reply docs/security-compliance/personal-data-location.md]: fixed

**Review by vinckr — APPROVED** [review]
👍

File: docs/security-compliance/personal-data-location.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+Ory Network handles this distinction for you transparently. You don't have to do anything to opt-in. Whatever data you query from
+wherever you are located, Ory Network will combine any globally replicated operational data with regional personal data for you
+automatically, reaching out behind the scenes to the region where that personal data is stored.
+
+## Selecting a storage location for personal data
+
+When you [create a new Ory Network project](https://console.ory.sh/projects/create) in the Ory Console, you are given a choice of
+personal data storage location for your new project. Any personal data of your users and identities will be stored just in that
+region , replicated within that region for availability and durability. Note that this choice is permanent for that project:
+moving personal data between regions is an administrative action, with which Ory Support is happy to assist enterprise customers.
```
Comment (aeneasr) [inline docs/security-compliance/personal-data-location.md:33]: Should we add a note that non-enterprise projects can choose one home region while enterprise can be global (multiple regions)?

**Comment by vinckr** [comment]
ready to merge @alnr ?

**Pull request merged**